onebusaway-webapp/src/main/resources/org/onebusaway/webapp/gwt/AgenciesMap.gwt.xml

```diff
@@ -43,5 +43,9 @@
 
 	<!-- Specify the app entry point class -->
 	<entry-point class='org.onebusaway.webapp.gwt.agencies_map.AgenciesMap' />
+	
+	<!-- Availabe Locales -->
+	<extend-property name="locale" values="en"/>
+	<extend-property name="locale" values="es"/>
```

Comment:
For GWT components, it is necessary to add all supported languages on some gwt.xml.

Not sure if you only need to add one or all files.